examples/z_sub_shm.c

```diff
@@ -36,7 +36,7 @@ void data_handler(const z_loaned_sample_t *sample, void *arg) {
 #if defined(SHARED_MEMORY) && defined(UNSTABLE)
     char *payload_type = "RAW";
     {
-        z_loaned_shm_t *shm = NULL;
+        const z_loaned_shm_t *shm = NULL;
```

Comment:
This shouldn't be `const`, isn't it? We are deserializing into it.

Reply:
We are getting pointer's value, not deserializing into it (as it is pointing to NLL)